src/model_instance.rs

```diff
@@ -407,7 +407,7 @@ impl ModelInstance {
 		/*! A method that enables updating hyperparameters of an existing (pre-loaded) model.
 		Currently limited to the most commonly used hyperparameters: ffm_learning_rate, ffm_power_t, power_t, learning_rate. */
 		
-		println!("Replacing initial regressor's hyperparameters from the command line ..");
+//		println!("Replacing initial regressor's hyperparameters from the command line ..");
```

Comment:
this is also important to add and have visibility of

Reply:
I actually think this one is redundant. It will print either way, even if there are no replacements, it seems

Reply:
ok so lets add a print to those if cases because we need to understand which values were changed